core/templates/pages/exploration-player-page/templates/information-card-modal.directive.html

```diff
@@ -115,10 +115,10 @@ <h2 ng-style="titleWrapper()" class="oppia-info-card-title"><[explorationTitle]>
   .oppia-info-card-bottom-row {
     line-height: 1.5;
   }
-  .flex-40 {
+  .tag-spacing {
```

Comment:
Instead, would it be possible to use `pb-1` or similar from [here](https://getbootstrap.com/docs/4.4/utilities/spacing/).

Reply:
It adds to much padding/margin than required.